tests/unittests/Makefile

```diff
@@ -4,7 +4,14 @@
 
 include $(CLEAR_VARS)
 
+TESTS_TO_BUILD := Foundation
+
+# Makefile Entrypoint: for running `make` without an argument.
+.PHONY: all
+all: $(foreach t,$(TESTS_TO_BUILD),$(t).UnitTests.exe)
+
```

Comment:
Thanks for taking care of this for me, it would've been the next thing to hit after looking at the current failures.